tests/flakes.sh

```diff
@@ -104,6 +104,13 @@ EOF
 # Test 'nix flake info'.
 nix flake info --flake-registry $registry flake1 | grep -q 'ID: *flake1'
 
+# Test 'nix flake info' on a local flake. This generates a lockfile as
```

Comment:
I really don't think `nix flake info` should have this kind of side effect. Can you explain why you do?
Same for every other `nix flake ..` where `.. != update`.

Reply:
It no longer does since `getFlake` doesn't need a lockfile. However, `nix flake deps` does build a lockfile.